libs/judicial-system/formatters/src/lib/formatters.ts

```diff
@@ -299,3 +300,38 @@ export function formatGender(gender?: CaseGender): string {
       return 'Kynsegin/Annað'
   }
 }
+
+export function formatGenderPronouns(gender?: CaseGender): string {
+  switch (gender) {
+    case CaseGender.MALE:
+      return 'hann'
+    case CaseGender.FEMALE:
+      return 'hún'
+    case CaseGender.OTHER:
+    default:
+      return 'hán'
+  }
+}
+
+export function formatAppeal(
+  appealDecision: CaseAppealDecision | undefined,
+  stakeholder: string,
+  stakeholderGender?: CaseGender,
```

Comment:
Consider simplifying this method, by defaulting to CaseGender.MALE. That way, there is no need to check below if stakeholderGender is set.

Reply:
Ah! Thank you.. Done!